include/sycl_ext_complex.hpp

```diff
@@ -256,19 +256,7 @@ template<class T> complex<T> tanh (const complex<T>&);
 
 _SYCL_EXT_CPLX_BEGIN_NAMESPACE_STD
 
-using std::enable_if;
-using std::integral_constant;
-using std::is_floating_point;
-using std::is_integral;
-using std::is_same;
-
-using std::basic_istream;
-using std::basic_ostream;
-using std::basic_ostringstream;
-
-using std::declval;
-
-template <bool _Val> using _BoolConstant = integral_constant<bool, _Val>;
+template <bool _Val> using _BoolConstant = std::integral_constant<bool, _Val>;
 
 template <class _Tp, class _Up>
 using _IsNotSame = _BoolConstant<!__is_same(_Tp, _Up)>;
```

Comment:
Not really related to this PR, but do you know from where this `__is_same` is coming from? Should we use `std::is_same`? And maybe inline `_BoolConstant` as we don't use it elsewhere

Reply:
So I did a quick investigation, `__is_same` is a clang defined operation, I am unsure if other compilers have it as well. I have replaced it with `std::is_same` for consistency.


> And maybe inline `_BoolConstant` as we don't use it elsewhere


Thats a great idea! `_BoolConstant` is only used in `_IsNotSame` and `_IsNotSame` is also only used once as well. So I have just inlined `_IsNotSame` and removed both typedefs.